### ISSUE TITLE
verify/scrape new support critera publications from health ministry page

### ISSUE THREAD

**Issue by noamoss** [OpenBudget/BudgetKey#347]
https://www.health.gov.il/Subjects/Finance/Mtmicha/Pages/default.aspx:

A. Scan the "פרסומים אחרונים" section:

![image](https://user-images.githubusercontent.com/3581741/55034612-801aed00-501e-11e9-843c-f3d9f8b6f00a.png)

For each item verify if it was already scraped from this source or from the justice ministry (via internal db query, as long as titles are ~80% similar no need to continue).

B. If the item is not included in our DB yet, add the title, the file and the date it was found in the db.

**Comment by akariv** [comment]
We need to talk about this source - it's very problematic.

**Comment by noamoss** [comment]
@akariv 
bringing it up again, as it seems like we did not talk about it eventually. 
Can you summarize here, or prefer to talk on Sunday?

**Comment by akariv** [comment]
We did speak about it - since it's a links only source (no actual info or dates), we said we'd scrape this and have the mechanism described in #408 to support it. 
Makes sense to you @noamoss ?

**Comment by akariv** [comment]
And the comment above is actually relevant to #348, not to here.

**Comment by akariv** [comment]
Anyway, this issues seems irrelevant as all the interesting info in the 'last publications' box also appears in the main text of the page:
<img width="781" alt="Screen Shot 2019-11-10 at 14 32 16" src="https://user-images.githubusercontent.com/265333/68544005-0b3ace80-03c7-11ea-831d-1177f523c314.png">

**Comment by akariv** [comment]
(so basically a duplicate of #348)

**Comment by noamoss** [comment]
@akariv indeed, but for me these are two different checks later.

**Comment by akariv** [comment]
What checks?

**Comment by noamoss** [comment]
forget it, closing.
